tests/integration/logging/logging.spec.js

```diff
@@ -0,0 +1,35 @@
+const utils = require('@utils');
+const moment = require('moment');
+
+const validLog = (line, before, after) => {
+  const [date, action] = line.split(/\s/);
+  console.log(date, before.toISOString(), after.toISOString());
```

Comment:
Was this left in by accident?

Reply:
yes, sorry! I kept measuring the necessary buffer to never get bad intervals.